modules/ROOT/pages/amazon/amazon-s3-connector-reference.adoc

```diff
@@ -1667,82 +4852,314 @@ Sets the given bucket's website configuration. This operation requires the PutBu
 * <<config>>
 
 ==== Throws
-See <<exceptionslist1>>.
-
+* S3:INVALID_RANGE
+* S3:REDIRECT
+* S3:SERVICE_UNAVAILABLE
+* S3:REQUEST_TIME_TOO_SKEWED
+* S3:INVALID_STORAGE_CLASS
+* S3:MAX_MESSAGE_LENGTH_EXCEEDED
+* S3:REQUEST_IS_NOT_MULTI_PART_CONTENT
+* S3:UNEXPECTED_CONTENT
+* S3:NO_SUCH_BUCKET
+* S3:ENTITY_TOO_SMALL
+* S3:INVALID_SOAP_REQUEST
+* S3:INVALID_PART
+* S3:METADATA_TOO_LARGE
+* S3:NO_SUCH_VERSION
+* S3:PRECONDITION_FAILED
+* S3:BUCKET_ALREADY_EXISTS
+* S3:MISSING_SECURITY_ELEMENT
+* S3:AMBIGUOUS_GRANT_BY_EMAIL_ADDRESS
+* S3:INVALID_URI
+* S3:NOT_SIGNED_UP
+* S3:INVALID_ADDRESSING_HEADER
+* S3:MAX_POST_PRE_DATA_LENGTH_EXCEEDED_ERROR
+* S3:ILLEGAL_VERSIONING_CONFIGURATION_EXCEPTION
+* S3:INVALID_PAYER
+* S3:INVALID_SECURITY
+* S3:NO_SUCH_BUCKET_POLICY
+* S3:OPERATION_ABORTED
+* S3:INVALID_ACCESS_KEY_ID
+* S3:INVALID_LOCATION_CONSTRAINT
+* S3:INLINE_DATA_TOO_LARGE
+* S3:SIGNATURE_DOES_NOT_MATCH
+* S3:UNRESOLVABLE_GRANT_BY_EMAIL_ADDRESS
+* S3:NO_SUCH_LIFECYCLE_CONFIGURATION
+* S3:KEY_TOO_LONG
+* S3:INVALID_BUCKET_NAME
+* S3:INVALID_TOKEN
+* S3:TEMPORARY_REDIRECT
+* S3:BUCKET_NOT_EMPTY
+* S3:MALFORMED_POST_REQUEST
+* S3:BAD_DIGEST
+* S3:CROSS_LOCATION_LOGGING_PROHIBITED
+* S3:REQUEST_TIMEOUT
+* S3:TOO_MANY_BUCKETS
+* S3:NO_SUCH_KEY
+* S3:MISSING_ATTACHMENT
+* S3:INVALID_BUCKET_STATE
+* S3:RETRY_EXHAUSTED
+* S3:INVALID_PART_ORDER
+* S3:INVALID_RESPONSE_PRESIGNEDURL_MALFORMED
+* S3:INVALID_POLICY_DOCUMENT
+* S3:MALFORMED_XML
+* S3:ACCOUNT_PROBLEM
+* S3:USER_KEY_MUST_BE_SPECIFIED
+* S3:MISSING_CONTENT_LENGTH
+* S3:INVALID_DIGEST
+* S3:ACCESS_DENIED
+* S3:MISSING_REQUEST_BODY_ERROR
+* S3:REQUEST_TORRENT_OF_BUCKET_ERROR
+* S3:TOKEN_REFRESH_REQUIRED
+* S3:RESTORE_ALREADY_IN_PROGRESS
+* S3:NO_LOGGING_STATUS_FOR_KEY
+* S3:INVALID_OBJECT_STATE
+* S3:CONNECTIVITY
+* S3:INVALID_ENCRYPTION_ALGORITHM_ERROR
+* S3:PERMANENT_REDIRECT
+* S3:INVALID_REQUEST
+* S3:INCORRECT_NUMBER_OF_FILES_IN_POST_REQUEST
+* S3:UNKNOWN
+* S3:MALFORMED_ACL_ERROR
+* S3:CREDENTIALS_NOT_SUPPORTED
+* S3:NOT_IMPLEMENTED
+* S3:INVALID_TARGET_BUCKET_FOR_LOGGING
+* S3:INVALID_ARGUMENT
+* S3:NO_SUCH_UPLOAD
+* S3:INTERNAL_ERROR
+* S3:SLOW_DOWN
+* S3:METHOD_NOT_ALLOWED
+* S3:ENTITY_TOO_LARGE
+* S3:EXPIRED_TOKEN
+* S3:MISSING_SECURITY_HEADER
+* S3:BUCKET_ALREADY_OWNED_BY_YOU
+* S3:INCOMPLETE_BODY
 
 [[setObjectAcl]]
-=== Set Object ACL
+=== Set Object Acl
```

Comment:
```suggestion
=== Set Object ACL
```

Reply:
This might be the actual operation name so I am leaving as is.